### PR TITLE
use more thread local static metrics in `src/coprocessor/metrics.rs`

### PR DESCRIPTION
**Pull request by Renkai**
UCP #5708
### What problem does this PR solve?

Issue Number: part of https://github.com/tikv/tikv/issues/5708

### What is changed and how it works?

use more thread local static metrics 

### Check List 

Tests 

- Unit test

### Release note

### CODE REVIEW

**Comment by Renkai** [comment]
@sre-bot /release make dist_release

**Comment by Renkai** [comment]
@sre-bot /bench +sysbench +tpcc +tpch

**Comment by Renkai** [comment]
@sre-bot /run-all-tests

File: src/coprocessor/tracker.rs [hunk 1]
```diff
@@ -211,89 +211,85 @@ impl Tracker {
...
-                .inc_by(self.total_perf_stats.0.encrypt_data_nanos as i64);
-
-            cop_metrics
-                .local_copr_rocksdb_perf_counter
-                .with_label_values(&[self.req_ctx.tag, "decrypt_data_nanos"])
-                .inc_by(self.total_perf_stats.0.decrypt_data_nanos as i64);
-        });
-
-        tls_collect_scan_details(self.req_ctx.tag, &total_storage_stats);
+        let req_tag = match self.req_ctx.tag {
```
Comment (breeswish) [inline src/coprocessor/tracker.rs:214]: req_ctx.tag can be `ReqTag` to totally avoid possibility of panic.

**Pull request merged**